### PR TITLE
No jira: add avn pallet benchmarks

### PR DESCRIPTION
**Pull request by MBrozhko34**
## Proposed changes

Add benchmarks to the avn pallet

## Type of change/Merge

🚨What type of change is this PR? </br>
_Put an `x` in the boxes that apply_

- [ ] Release 
  - [ ] Increase versions 
  - [x] Baseline tests passed  
  - Release type:
    - [ ] Major release 
    - [ ] Minor release 
    - [ ] Patch release 

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [x ] You describe the purpose of the PR, e.g.:
  - What does it do?
  - Highlight what important points reviewers should know about;
  - Indicates if there is something left for follow-up PRs.
- [ ] Documentation updated
- [x] Business logic tested successfully
- [ ] Verify First, Write Last: In Substrate development, it is important that you always ensure preconditions are met and return errors at the beginning. After these checks have completed, then you may begin the function's computation.

### CODE REVIEW

File: pallets/avn/src/benchmarking.rs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+//! # Ethereum events pallet
+// Copyright 2022 Aventus Systems (UK) Ltd.
```
Comment (nahuseyoum) [inline pallets/avn/src/benchmarking.rs:2]:
```suggestion
// Copyright 2023 Aventus Systems (UK) Ltd.
```

File: pallets/avn/src/benchmarking.rs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+//! ethereum-events pallet benchmarking.
```
Comment (nahuseyoum) [inline pallets/avn/src/benchmarking.rs:4]:
```suggestion
//! avn pallet benchmarking.
```

File: pallets/avn/src/benchmarking.rs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
+use frame_system::{EventRecord, RawOrigin};
+
+benchmarks! {
+    set_bridge_contract {
+        let contract_address = H160::from([1; 20]);
+    }: set_bridge_contract(RawOrigin::Root, contract_address.clone())
+    verify {
+        assert!(<AvnBridgeContractAddress<T>>::get() == contract_address);
```
Comment (nahuseyoum) [inline pallets/avn/src/benchmarking.rs:18]: You can also assert that an event has been emitted, this way your `EventRecord` import can be used :)

**Review by nahuseyoum — COMMENTED** [review]
Don't we need an additional change to the runtime to update the `define_benchmarks!` macro?
And can we test by running the actual benchmarks?

**Review by alexpArtos — APPROVED** [review]

**Pull request merged**